src/aat/resources/features/F-042/F-042.feature

```diff
@@ -0,0 +1,82 @@
+@F-042
+Feature: F-042: Trigger "aboutToStart" event as a Case worker
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+  @S-243
+  Scenario: Trigger the aboutToStart callback event for a caseworker for a new case which has not been started yet.
+    Given a user with [an active profile in CCD]
+    And a case that has just been created as in [Caseworker1_Full_Case]
+    When a request is prepared with appropriate values
+    And the request [is prepared with a valid User ID, Jurisdiction, Case ID, Case Type ID and Event Trigger ID]
```

Comment:
refer to the case id as "the case id just created"

Reply:
Updated the request spec.